### ISSUE TITLE
Makima does not recognize right joystick movement using Google Stadia Controller

### ISSUE THREAD

**Issue by pogmommy** [cyber-sushi/makima#26]
Hi, firstly thank you for your work on this astounding project.

I am trying to use Makima with my Google Stadia Controllers (Rev. A), and while all other inputs appear to work great, the right joystick's axis input does not appear to be recognized in Makima.

Troubleshooting steps I've taken:

- confirmed that `evtest` does recognize the input
- tested two other Stadia Controllers with no success
- tested the same config on a Dualshock 4 controller which worked as expected
- tried connecting the Stadia Controller via cable and bluetooth, the issue is present regardless of connection method
- using different mappings on the same input (`scroll` and `cursor` both work on `LSTICK`, but neither work on `RSTICK`)
- adjusting `RSTICK_SENSITIVITY` and `RSTICK_DEADZONE`, neither of which made any difference

[Here](https://github.com/user-attachments/files/18174723/evtest_stadia.log) is the output of the following inputs in `evtest`:

- Right Stick
  - Up
  - Down
  - Left
  - Right
- Left Stick
  - Up
  - Down
  - Left
  - Right

And [here](https://github.com/user-attachments/files/18174741/Google.LLC.Stadia.Controller.rev.A.toml.txt) is my config (Github would not let me upload the file with the .toml file extension)

My system is a Debian Unstable machine running version 0.9.5 of Makima.

Please let me know if there's any additional information I can provide or troubleshooting I can perform that would be helpful.

**Comment by cyber-sushi** [comment]
Hello and thank you so much for the detailed report!

The issue seems to be that while other controllers emit `ABS_X`/`ABS_Y` events for the left stick and `ABS_RX`/`ABS_RY` for the right stick, the Stadia controller emits `ABS_Z`/`ABS_RZ` for the right stick. `Z` events are usually designed for a controller's triggers, but according to the notes I took when I first made Makima, a Stadia controller emits `ABS_GAS`/`ABS_BRAKE` when pressing the triggers (plus `BTN_TRIGGER_HAPPY3`/`BTN_TRIGGER_HAPPY4`, but that doesn't matter here).

The solution should be pretty easy, I'm just gonna add a `STADIA` parameter under `[settings]` that defaults to false, and if it's set to true Makima is gonna read `Z` events as the right stick. It should be in the next release.

Thank you again!
